.github/workflows/label-schema.yml

```diff
@@ -11,4 +11,6 @@ jobs:
       uses: rtCamp/action-slack-notify@v2
       continue-on-error: true
       env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_SCHEMA_CHANGE }}
+        SLACK_FOOTER: $PR_TITLE
```

Comment:
Correct me if I am wrong, but at a glance, I'm skeptical that `$PR_TITLE` will be propagated to `SLACK_FOOTER`. I would have expected to see something more like...

```suggestion
        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_SCHEMA_CHANGE }}
        SLACK_FOOTER: ${{ github.event.pull_request.title }}
```

Has this been tested?

Reply:
Thanks for the review @flrgh. The suggested approach exposes a command injection threat https://stackoverflow.com/a/76679447. I will test this out and let you know my findings. 

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions

Reply:
I don't think there is a command injection, because this action isn't invoking any command.

Reply:
Makes sense, I have updated the PR.